main.go

```diff
@@ -92,15 +92,13 @@ func main() {
 	}
 
 	zeroLogger.Trace().Msg("Initializing API server")
-	go func() {
-		err = srv.ListenAndServe(serverCtx, logger)
-		if err != nil {
-			zeroLogger.Error().Err(err).Msg("Failed to initialize API server")
-			raven.CaptureErrorAndWait(err, nil)
-			logger.Panic(err)
-			return
-		}
-	}()
+	err = srv.ListenAndServe(serverCtx, logger)
```

Comment:
I don't understand this diff because the existing code here in `master` is the same as what is claimed in this diff to be added. I'm chalking it up to getting confused by rebase but I'm noting it here in case anyone else thinks "wait, is the server in a goroutine in prod?", as I did. It is not.
https://github.com/brave-intl/challenge-bypass-server/blob/master/main.go#L95